docs/plugins/plugin-development-guide/naming.rst

```diff
@@ -0,0 +1,59 @@
+Naming changes
+--------------
+
+``PluginSkeleton`` provides some default classes and configurations. However, they must have some default values and names that should be changed
+to reflect your plugin functionality. Basing on the vendor and plugin names established above, these are the changes that should be made:
+
+* In ``composer.json``:
+
+    * ``sylius/plugin-skeleton`` -> ``iron-man/sylius-product-on-demand-plugin``
+
+    * ``Acme example plugin for Sylius.`` -> ``Plugin allowing to mark product variants as available on demand in Sylius.`` (or sth similar)
+
+    * ``Acme\\SyliusExamplePlugin\\`` -> ``IronMan\\SyliusProductOnDemandPlugin\\`` (the same changes should be done in namespaces in ``src/`` directory
+
+    * ``Tests\\Acme\\SyliusExamplePlugin\\`` -> ``Tests\\IronMan\\SyliusProductOnDemandPlugin\\`` (the same changes should be done in namespaces in ``tests/`` directory
+
+* ``AcmeSyliusExamplePlugin`` should be renamed to ``IronManSyliusProductOnDemandPlugin``
```

Comment:
why some of these points are tabbed and some are not?

Reply:
Some of them are sub-points for other points (e.g. "In `composer.json`" have few of them, as there are few things that need to be changed in `composer.json` file)